### ISSUE TITLE
Unable to complete Paladin Redemption quest: The Tome of Divinity/

### ISSUE THREAD

**Issue by joshthetechi** [OmegaWoWeu/Decades#83]
Description: Unable to complete the quest for the Paladin Redemption spell

How to reproduce bug: I have no quest giver in the Pally area of SW

How it should work: The quest giver should be me a book and sends me out near the abbey

Screenshots:
![image](https://user-images.githubusercontent.com/47923151/53288663-1c6c7e00-3759-11e9-8ab0-5ad7eb6595a7.png)

Relevant AddOns used:NA

Source: Character "HoldmeBeers

**Comment by mclgv** [comment]
Do u wanna do Resurrection spell quest?

**Comment by joshthetechi** [comment]
I would love to

**Comment by blzee** [comment]
Still having the issue?

**Comment by joshthetechi** [comment]
I am

**Comment by mclgv** [comment]
If its solved i dont know but First Pala trainer at goldshire tells u that visit Durothain Rall(whatever) in Cathedral, it gives u the q and directs u to go place, use swastika and ress the dorf if this quest does not appear. I dont know situation i did 2 times on different chars.

**Comment by joshthetechi** [comment]
I believe the trainer gives you a time that actually gives you the quest.
At the time it was bugged and the tome when right clicking on it did not
give me the quest. I drop/destroyed the tome thinking I could pick it back
up. The trainer in SW does not seem to want to give me the quest again. Is
it possible to just give me the spell or are you able to reset so I can
pick up the quest again?

**Comment by joshthetechi** [comment]
I will check goldshire again and provide you with and update.

**Comment by joshthetechi** [comment]
I went to the first paladin trainer in Goldshire like you mentioned. I have
show Low level quests turned on, but he did not have a quest for me...See
attached screenshot.

I then went to the Cathedral as mentioned as the next step with Duthorian
Rall, who also didn't have a quest for me...See attached screenshot.

I also took a screenshot of my current quest log to show I don't have the
quest needed.
